src/frontend/tsx/index.tsx

```diff
@@ -1,13 +1,17 @@
 import React from 'react'
 import '../css/tabs.css'
 import '../css/text.css'
+import '../css/boards.css'
+import '../css/pieces.css'
 import '../css/popups.css'
 import '../css/buttons.css'
 import '../css/general.css'
 import '../css/graphics.css'
 import '../css/textboxes.css'
 import '../css/responsive.css'
 import '../css/chessground.css'
+import 'chessground/assets/chessground.base.css' // Import Chessground style sheets
+import 'chessground/assets/chessground.brown.css'
```

Comment:
We don't need this one anymore, right? Since we're going to use our own brown as the default.